ops/ansible/playbooks-ccs/build_bfd-server.yml

```
@@ -2,66 +2,48 @@
 ##
 # Configures the ETL Server and data-pipeline app for CCS env
 ##
-
 - name: Configure BFD Data Server
   hosts: all
   become: true
   remote_user: ec2-user
   gather_facts: no
   vars:
     ansible_ssh_pipelining: no
-    bfd_server_dir: '/usr/local/bfd-server'
+    data_server_dir: '/usr/local/bfd-server'
     env: "test"
 
   tasks:
-    - name: Include global variables
-      include_vars:
-        file: vars/000_cross_env_vars.yml
+    - name: Download BFD Repository
+      git:
+        repo: https://github.com/CMSgov/beneficiary-fhir-data.git
+        dest: /beneficiary-fhir-data
+        # use hash from `git_commit` when on stable master branch, otherwise use git_branch name
+        version: "{{ git_branch if git_branch != 'master' else git_commit }}"
```

Considering adjusting the conditional here...
```suggestion
        version: "{{ git_commit if git_branch == 'master' else git_branch }}"
```